ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -41,7 +43,14 @@ component:
   valve: "Component which meters the flow of water within a system or device."
   vane: "Component for guiding media flow."
   wheel: "Component used for transfer of heat from incoming to outgoing air streams. "
-descriptor:
+descriptor: 
+  emergency : "a situation that poses an immediate risk to health, life property,  or environment."
+  scene : "a combination of objects or events in view or happening of an event that attracts attention."
+  leak : "Media leaving the system abnormally."
+  detection : "Process of identifying conditions."
+  fire : "a potential to cause physical damage through burning"
+  suppression : "a range of firefighting tactics used to suppress fire"
+  Photocatalyst : "a material that uses absorbed light energy (usually UV) to drive desired reactions that would not otherwise proceed under ambient conditions"
```

Comment:
can you please make it low capital letter: Photocatalyst ==> photocatalyst.

Reply:
Ok, changes commited.